
Of course the shared library needs PIC - imported from ROCK Linux.

  - Rene Rebe <reneÂ@exactcode.d>

diff -Nru cracklib,2.7-orig/cracklib/Makefile cracklib,2.7/cracklib/Makefile
--- cracklib,2.7-orig/cracklib/Makefile	2004-11-06 09:48:53.010325096 +0100
+++ cracklib,2.7/cracklib/Makefile	2004-11-06 09:49:08.042039928 +0100
@@ -8,7 +8,7 @@
 
 LIB=	libcrack.a
 OBJ=	fascist.o packlib.o rules.o stringlib.o
-CFLAGS= -O -I../cracklib -DIN_CRACKLIB
+CFLAGS= -O -I../cracklib -DIN_CRACKLIB -fPIC
 
 $(LIB):	$(OBJ)
 	ar rv $(LIB) $?
